src/test/java/ai/zuva/ZdaiApiClientTest.java

```diff
@@ -0,0 +1,42 @@
+package ai.zuva;
+
+import com.github.tomakehurst.wiremock.junit5.WireMockRuntimeInfo;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Objects;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@WireMockTest
+class ZdaiApiClientTest {
```

Comment:
No, I started it and just never opened the file again. I'll delete it for this PR, since all the methods in ZdaiApiClient are tested transitively by the existing tests.